### DISCUSSION TITLE
主应用在window上挂公共组件，子应用拿到，无法渲染前端ui组件，主子应用全部是antdv

### DISCUSSION THREAD

**Discussion by nuote** [umijs/qiankun#1973] [Q&A]
主子应用都是ant-design-vue + vue3 有其他父子应用共享组件的方案么

**Comment by sunlianglife** [comment]
有方案了么

**Comment by gujimiss** [comment]
我也遇到这样的问题，主应用直接把组件通过pops传给子应用，子应用通过vue.components方法注册全局组件，渲染是能渲染出来，但是会报错，[Vue warn]: $attrs is readonly. 这个错误信息。
